force-app/main/default/lwc/accountModelSettings/accountModelSettings.html

```diff
@@ -7,12 +7,81 @@
                 onsettingssaving={handleSettingsSaving}
                 onsettingssavecompleted={handleSettingsSaveCompleted}
             >
-                <!-- Body of Account Model Settings page (populates Slot in child component) -->
-                <lightning-button
-                    label="Test Button"
-                    disabled={affordancesDisabled}
-                ></lightning-button>
             </c-settings-save-canvas>
+
+            <!-- Body of Account Model Settings page (populates Slot in child component) -->
+            <template if:true={accountModelSettingsViewModel}>
+                <div class="slds-grid slds-grid_vertical-align-start slds-wrap">
+                    <!-- Default Account Record Type Setting -->
+                    <div class="slds-col slds-size_2-of-3 slds-p-bottom_x-large">
+                        <div class="slds-text-title slds-text-color_default slds-p-left_small">
```

Comment:
Let's bump the font size up on this a little. Let's try using slds-text-title_bold